### DISCUSSION TITLE
Lodestar Standup 2024-33 (August 27, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7040] [Standups]
### Meeting Info
[August 27, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/aug-27-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Electra rebase/merge
- Open Discussion
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
EF Devops looking for a build for the latest [alpha4 spec](https://github.com/ethereum/consensus-specs/releases/tag/v1.5.0-alpha.4) by Wednesday if possible. Will chat about anything missing to pass these tests.

**Reply by philknows** [reply]
https://notes.ethereum.org/@ethpandaops/pectra-devnet-3

**Comment by philknows** [comment]
Review Lodestar user survey with Andrew to go over responses and data representation.

**Reply by twoeths** [reply]
I'm curious to know the setup that gave gave bad performance of lodestar
same to ProbeLab https://github.com/ChainSafe/lodestar/issues/7046

**Comment by matthewkeil** [comment]
@acolytec3 was asking in #7052 about semver of internal packages.  Their project depends on `@lodestar/types` and broke with our types update when they updated from 1.17 -> 1.21

I know we discussed separate version on a couple prior occasions so figured we could rehash that decision to see where we land this go around.

**Reply by philknows** [reply]
Light client, prover and types are likely the ones that we need to pay attention to for breaking changes.

**Comment by twoeths** [comment]
plan to merge electra to `unstable` to unblock other PRs

**Comment by acolytec3** [comment]
Here's the [glados](https://glados.ethdevops.io/) "explorer" (such as it is) for the Portal Network that we discussed.  Looks like it only reports on EL block history at the moment.

**Comment by philknows** [comment]
Notes posted: https://github.com/ChainSafe/lodestar/wiki/Lodestar-Planning-&-Standup-Meetings#august-27-2024-planning-and-standup-meeting
Closing for https://github.com/ChainSafe/lodestar/discussions/7054
